source/jormungandr/jormungandr/interfaces/v1/journey_common.py

```diff
@@ -68,6 +68,15 @@ def schema(self):
         return TypeSchema(type=str, metadata={'enum': ['arrival', 'departure'], 'default': 'departure'})
 
 
+def sort_regions(regions_list):
+    """
+    Sort the regions according to the instances_comparator (priority > is_free=False > is_free=True)
+    :param regions_list: possible regions for the journeys request
+    :return: sorted list according to the comparator
+    """
+    return sorted(regions_list, key=cmp_to_key(instances_comparator))
```

Comment:
We may also sort them alphabetically as a last criterion to have stable outputs (thinking of Artemis if we ever want to test it there).

Reply:
It may be a good idea indeed. However, it may also change some tests again... TBD with the architect